### PR TITLE
Kristiana tatarcuka

### PR DESCRIPTION
**Pull request by KristianaTR**
Changes made:

1. create ProductCard component
2. create ProductList component

### CODE REVIEW

File: src/Components/ProductCard/ProductCard.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+            <img 
+                src="https://images.demandware.net/dw/image/v2/BBBV_PRD/on/demandware.static/-/Sites-master-catalog/default/dw9f9383a1/images/2050000/2052276.jpg?sw=1200" 
+                alt="product img" 
+            />
+        </div>
+        <div className="ProductDataContainer">
+            <div className="ProductName">Stylish sofa</div>
+            <div className="ProductPrice">650 EUR</div>
+        </div>
+        <button className='ProductCardQuickviewButton'>Quickview</button>
```
Comment (lindalejiete) [inline src/Components/ProductCard/ProductCard.tsx:16]: I would suggest to import this as reusable component

File: src/Components/ProductList/ProductList.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import ProductCard from '../ProductCard/ProductCard';
+import './ProducrList.css';
+
+const ProductList = () => {
+  return (
+    <div className='ProductListWrapper'>
+        <ProductCard/>
```
Comment (lindalejiete) [inline src/Components/ProductList/ProductList.tsx:7]: As I understand, we will use dynamic properties, so one component is enough here

File: src/Components/ProductCard/ProductCard.tsx (hunk 1 above)
Comment (KristianaTR) [reply src/Components/ProductCard/ProductCard.tsx:16]: Good point. Will do that.

File: src/Components/ProductList/ProductList.tsx (hunk 2 above)
Comment (KristianaTR) [reply src/Components/ProductList/ProductList.tsx:7]: Yes, you are right. I will delete extra components when data.js is provided. 
It is duplicated for now just to have a visualization.

**Pull request merged**